cl/opinion_page/forms.py

```diff
@@ -158,81 +168,14 @@ class CourtUploadForm(forms.Form):
             }
         ),
     )
-    date_argued = forms.DateField(
-        label="Argued Date",
-        required=False,
-        widget=forms.TextInput(
-            attrs={
-                "class": "form-control datepicker",
-                "placeholder": "Argued Date",
-                "autocomplete": "off",
-            }
-        ),
-    )
-    date_reargued = forms.DateField(
-        label="Reargued Date",
+    download_url = forms.URLField(
+        validators=[URLValidator()],
+        label="Download Url",
```

Comment:
this field is now available in ME uploader - maybe tenn. but it wasn't /isnt in the current iteration.  can you make sure we dont alter the uploads for the other courts.